### DISCUSSION TITLE
Mixing this with pagination of diary

### DISCUSSION THREAD

**Discussion by BradKML** [YU000jp/logseq-plugin-single-journal#3] [Ideas]
It would be good to be able to add information for yesterday rather than today when something was missed. Would it be compatible?
Idea in question https://github.com/isosphere/logseq-journals-nextprev

**Comment by YU000jp** [comment]
It's difficult to understand that correctly.  Could you please explain it again?

**Reply by BradKML** [reply]
I checked that there is a calendar plugin that allows people to select missing days for writing journal (there are days when the app is not open and the journal function skips those days).

**Reply by YU000jp** [reply]
Is your request for an option to open yesterday's page instead of today's?

**Reply by BradKML** [reply]
Yeah, but the normal option is simply not there so I used the calendar plugin to work around it

**Reply by YU000jp** [reply]
Is your goal to create a journal page for yesterday that doesn't exist?

**Reply by BradKML** [reply]
Yeah

**Reply by YU000jp** [reply]
https://github.com/YU000jp/logseq-plugin-show-weekday-and-week-number
Use "Show weekday and week-number" plugin instead. A mini-calendar will appear with an indicator as to whether the journal exists.

**Reply by BradKML** [reply]
I am on this one atm since I also like to fill in previous days https://github.com/xyhp915/logseq-journals-calendar
P.S. Am-a journal template abuser https://github.com/xyhp915/logseq-journals-calendar/issues/44

**Reply by YU000jp** [reply]
It is a specification of Logseq that templates are not applied to past journals.

**Reply by YU000jp** [reply]
A new plugin, [Default Template plugin](https://github.com/YU000jp/logseq-plugin-default-template), has a journal template completion feature. Put the name of the template in the plugin settings. This will handle cases where the journal template is not applied even though it is a journal attribute.
